### PR TITLE
Rerouting

### PR DESCRIPTION
**Pull request by alexrpark**
Hi!

To test, go on branch rerouting. 

I made it follow the prototype and had the creation and updating of the list happen on /lists. 
I deleted the other views since we won't be needing those extra pages.

### CODE REVIEW

File: smart-cart/app/controllers/lists_controller.rb [hunk 1]
```diff
@@ -5,6 +5,8 @@ class ListsController < ApplicationController
   # GET /lists or /lists.json
   def index
     # @lists = List.all
+    @list = []
+    @current_list = $list
```
Comment (e33lin) [inline smart-cart/app/controllers/lists_controller.rb:9]: now that the app is sitting on a database, we should remove these and use the actual List resource to build out a list instead the arrays

**Review by e33lin — COMMENTED** [review]
tested and everything looks good to me! my only thing is that i think we intended the "create list page" task to rlly encompass full functionality of like adding items, removing, changing quantity, using database, etc. so thats why its a 5-pointer. but honestly i rlly like how this PR is LOL like it just sets us up to go ahead and build out the rest. so i propose:
1. draft this PR, build out the rest of the create list page and push
2. we open a second 3-pointer task to do the rest of the create list stuff and pass the second task off to someone else if you would like
what do you think?

File: smart-cart/app/controllers/lists_controller.rb (hunk 1 above)
Comment (jdexcond) [reply smart-cart/app/controllers/lists_controller.rb:9]: yeah i like that idea, i can help out with that second task

Comment (e33lin) [reply smart-cart/app/controllers/lists_controller.rb:9]: okay nice, @jdexcond i can pair with you on it if you get stuck or anything!!
@alexrpark if thats cool w you, i'll just approve this PR and make the second task

Comment (alexrpark) [reply smart-cart/app/controllers/lists_controller.rb:9]: @e33lin sounds good, I'll also help w the second part!

**Review by e33lin — APPROVED** [review]

**Pull request merged**